src/sass/app/websocket_pages.scss

```diff
@@ -300,3 +300,65 @@
         width: 280px !important;
     }
 }
+
+#paymentagent_withdrawal {
+    .wrapper-row-agent {
+        label {
+            padding-bottom: 10px;
+        }
+        .row-agent {
+            width: fit-content;
+
+            > div {
+                &:not(:first-child) {
+                    margin-top: 10px;
+                }
+                &:nth-child(2) {
+                    text-align: center;
+                }
+            }
+            .hint {
+                width: min-content;
+                line-height: 1.4;
+            }
+        }
+    }
+    .select-dropdown, #txtAgents, #txtAmount, .row-agent .hint, .row-agent > div:nth-child(2) {
```

Comment:
imo, its better to break line here?

Reply:
@cakasuma eslint gives error on multiline